packages/edit-site/src/components/page-templates/style.scss

```diff
@@ -37,6 +37,8 @@
 	&.is-viewtype-table {
```

Comment:
Do we still have these classes? I don't think they exist anymore no? you might want to rebase and adapt your code.